### PR TITLE
Build: Use a 6.0.200 SDK and macos workload P13

### PR DESCRIPTION
**Pull request by sandyarmstrong**
Preview 13 of the macos workload has several breaking changes, not all of them documented yet. This must be merged carefully.

### CODE REVIEW

File: Xwt.XamMac/Xwt.Mac/OutlineViewBackend.cs [hunk 1]
```diff
@@ -26,6 +26,8 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
+global using ObjCRuntime;
```
Comment (sandyarmstrong) [inline Xwt.XamMac/Xwt.Mac/OutlineViewBackend.cs:29]: In every other repo I've been actually adding `using ObjCRuntime;` in each file that needed it. Another option would be to allow implicit usings and specify it there. I have no preference. I did the `global using` shortcut here to save some time.

File: Xwt.XamMac/Xwt.Mac/PopoverBackend.cs [hunk 2]
```diff
@@ -305,7 +305,7 @@ public class NSAppearanceCustomizationPopover : NSPopover, INSAppearanceCustomiz
 			public NSAppearanceCustomizationPopover ()
 			{ }
 
-			protected NSAppearanceCustomizationPopover (IntPtr handle) : base (handle)
+			protected NSAppearanceCustomizationPopover (NativeHandle handle) : base (handle)
```
Comment (sandyarmstrong) [inline Xwt.XamMac/Xwt.Mac/PopoverBackend.cs:308]: These `NativeHandle` changes mean we cannot merge this until we're ready to build VSMac with a newer macos workload.

**Review by Therzok — APPROVED** [review]

**Review by sevoku — APPROVED** [review]

**Comment by sevoku** [comment]
@sandyarmstrong pls let me know when this is ready to be merged.

**Review by slluis — APPROVED** [review]

**Review by sevoku — APPROVED** [review]

**Comment by sandyarmstrong** [comment]
@sevoku @slluis I think we'll need to start pushing newer versions of xwt to nuget.org. Some Android-related components grab it via PackageReference.

File: Xwt.XamMac/Xwt.Mac/TextLayoutBackendHandler.cs [hunk 3]
```diff
@@ -260,8 +260,8 @@ public void Draw (CGContext ctx, CGColor foregroundColor, double x, double y)
 					TextLayout.AddTextContainer (TextContainer);
 					TextStorage.AddLayoutManager (TextLayout);
 
-					TextLayout.DrawBackgroundForGlyphRange (new NSRange(0, Text.Length), new CGPoint (x, y));
-					TextLayout.DrawGlyphsForGlyphRange (new NSRange(0, Text.Length), new CGPoint (x, y));
+					TextLayout.DrawBackground (new NSRange(0, Text.Length), new CGPoint (x, y));
+					TextLayout.DrawBackground (new NSRange(0, Text.Length), new CGPoint (x, y));
```
Comment (sandyarmstrong) [inline Xwt.XamMac/Xwt.Mac/TextLayoutBackendHandler.cs:264]: This should be `DrawGlyphs`, it's the cause of https://github.com/xamarin/vsmac/pull/5897#issuecomment-1043236008.

Comment (sandyarmstrong) [reply Xwt.XamMac/Xwt.Mac/TextLayoutBackendHandler.cs]: Fixed.

File: Xwt.XamMac/Xwt.Mac/OutlineViewBackend.cs (hunk 1 above)
Comment (Therzok) [reply Xwt.XamMac/Xwt.Mac/OutlineViewBackend.cs:29]: I am okay with global using here or at MSBuild / AssemblyInfo.cs level.

**Review by Therzok — APPROVED** [review]

**Pull request merged**